src/test/java/tds/exam/results/mappers/OpportunityMapperTest.java

```diff
@@ -0,0 +1,132 @@
+package tds.exam.results.mappers;
+
+import org.joda.time.Instant;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import tds.assessment.Assessment;
+import tds.assessment.Item;
+import tds.exam.Exam;
+import tds.exam.ExamAccommodation;
+import tds.exam.ExamItem;
+import tds.exam.ExamItemResponse;
+import tds.exam.ExamPage;
+import tds.exam.ExamSegment;
+import tds.exam.ExpandableExam;
+import tds.exam.results.trt.TDSReport;
+import tds.session.Session;
+
+import static io.github.benas.randombeans.api.EnhancedRandom.random;
+import static io.github.benas.randombeans.api.EnhancedRandom.randomListOf;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
doesn't look like this is needed.  Same throughout.  If the test isn't mocking let's leave this off for now.